src/flow_solver/flow_solver_cases/periodic_turbulence.cpp

```diff
@@ -544,7 +544,13 @@ double PeriodicTurbulence<dim, nstate>::get_strain_rate_tensor_based_dissipation
 }
 
 template<int dim, int nstate>
-double PeriodicTurbulence<dim, nstate>::get_numerical_entropy(
+double PeriodicTurbulence<dim, nstate>::get_numerical_entropy(const std::shared_ptr <DGBase<dim, double>> /*dg*/) const
```

Comment:
if this is a getter, you can remove the argument here

Reply:
or just remove the function since it doesn't look like its being called anywhere

Reply:
Per in-person discussion, this function is used in a test. I will resolve comments related to it.